azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -755,12 +768,6 @@ def remove_extension_slice(self, extension_name):
                 if os.path.exists(extension_slice_path):
                     self.stop_tracking_extension_cgroups(extension_name)
                     CGroupConfigurator._Impl.__cleanup_unit_file(extension_slice_path)
-                # stop the unit gracefully; the extensions slices will be removed from /sys/fs/cgroup path
-                try:
-                    logger.info("Executing systemctl stop {0}".format(extension_slice_name))
-                    shellutil.run_command(["systemctl", "stop", extension_slice_name])
-                except Exception as exception:
-                    _log_cgroup_warning("systemctl stop failed (remove slice): {0}", ustr(exception))
```

Comment:
It's not clear to me why this code was removed... What is the reasoning here?

Reply:
Regularly, it's the system that cleans up these slices if the croups are not active. 

This run is to remove the slices from /sys/fs/cgroup explicitly if still present. This becomes flaky and not working all the time and also, redhat and centos complaining about access denied while running this command. So, I decided to remove and relay on OS to preform cleanup.